.github/actions/nm-get-docker-tags/action.yml

```diff
@@ -22,15 +22,15 @@ runs:
   steps:
   - id: tags
     run: |
-      BUILD_VERSION=`echo "${{ inputs.wheel }}" | cut -d'-' -f2`
+      BUILD_VERSION=`echo "${{ inputs.whl }}" | cut -d'-' -f2`
       if [[ "${{ inputs.wf_category }}" == "RELEASE" ]]; then
-          TAG="v${build_version}"
+          TAG="v${BUILD_VERSION}"
           EXTRA_TAG=latest
       else
-          TAG=`echo "${build_version}" | cut -d'.' -f4`
+          TAG=`echo "${BUILD_VERSION}" | cut -d'.' -f4`
           EXTRA_TAG=nightly
       fi
-      if [[ "${{ inputs.wheel }}" == "latest" ]]; then
+      if [[ "${{ inputs.whl }}" == "latest" ]]; then
```

Comment:
why is this statement necessary?  it seems that line 25 would have already set `BUILD_VERSION="latest"`

Reply:
This is in the case when the whl is passed in as 'latest' and line 25 doesn't extract the expected version so we need to make sure build_version is set to "latest".